sphinx/source/governance.rst

```diff
@@ -3,7 +3,7 @@ Governance
 
 A trusted set of members is in charge of governing a given CCF network. For transparency and auditability, all governance operations are recorded in plaintext in the ledger.
 
-One member (proposer) can submit a new proposal. Once they have done this, other members can vote for the proposal using its unique proposal ID. Proposals are executed once a quorum of members have accepted it.
+One member (proposer) can submit a new proposal. Once they have done this, other members can vote for the proposal using its unique proposal ID. Proposals are executed once a :term:`Quorum` of members have accepted it.
```

Comment:
Minor: Does `quorum` work here, or do we need to use case-matching `Quorum`? If we need to case-match, I think the glossary entry should be lower case - this explains our usage of the generic term `quorum`, but is not a Proper Noun.